web/src/containers/ScheduleSummary.js

```diff
@@ -16,29 +16,42 @@ const ScheduleSummary = ({ activities }) => (
             {t('scheduleSummary.noDataMessage')}
           </div>
         ) : (
-          activities.map(({ name: activityName, milestones }, i) => (
-            <table key={activityName} className="table-cms ds-u-margin-top--0">
-              <thead>
-                <tr>
-                  <th colSpan="2" className="ds-u-font-weight--bold">
-                    Activity {i + 1}: {activityName}
-                  </th>
-                </tr>
-              </thead>
-              <tbody>
-                {milestones.map(({ end, name: milestoneName, start }) => (
+          activities.map(
+            ({ end, name: activityName, milestones, start }, i) => (
+              <table
+                key={activityName}
+                className="table-cms ds-u-margin-top--0"
```

Comment:
This will change with my open tables PR -- should that one get merged first and merged into this, or vice versa?

Reply:
Hmm.  How much of the table will need to change, you think?  If it's just changing this class name, then it doesn't seem like it'd matter much. But if it's a big change to the table, then I think merging this one first makes more sense - to keep the contents of the PR all related.

Reply:
Sorry I missed this! It'll change a few classnames, but nothing super major. Totally up to you!

Reply:
For the sake of not adding work to the tables PR, let's merge that one first! I suspect changing class names here will be easier.  😄